### ISSUE TITLE
Issues with package versions

### ISSUE THREAD

**Issue by gquipster** [lstein/Perl-GD#30]
Hi,

Despite having updated to latest version of GD I still get listed in the CPAN tool:

`GD::Polygon                   undef       0.2  RURBAN/GD-2.69.tar.gz`

The problem appears to be that the GD::Polygon package itself has no version, but the GD::Polyline package has a version of "0.2" and then overrides the GD::Polygon package

```
package GD::Polyline;

############################################################################
#
# GD::Polyline
#
############################################################################
#
# What's this?  A class with nothing but a $VERSION and @ISA?
# Below, this module overrides and adds several modules to 
# the parent class, GD::Polygon.  Those updated/new methods 
# act on polygons and polylines, and sometimes those behaviours
# vary slightly based on whether the object is a polygon or polyline.
#

use vars qw($VERSION @ISA);
$VERSION = "0.2";
@ISA = qw(GD::Polygon);


package GD::Polygon;

############################################################################
#
# new methods on GD::Polygon
#
############################################################################
```

I think the fix is to either remove the version or to align it with the main package version?

Thanks in advance

**Comment by rurban** [comment]
I think adding the version will fix it

**Comment by rurban** [comment]
Fixed with 2.70
